### PR TITLE
fix(plugin/microsoft-teams): incorrect markdown rendering

### PR DESCRIPTION
**Pull request by mathieubergeron**
# What Changed

Changes in this PR are related to the microsoft-teams plugin, and the following issue: https://github.com/intuit/auto/issues/2020.

I removed the part that escaped characters using the `jsesc` library.

Note that I also removed some of the unit tests, which was either duplicated or not useful. I will leave comments about those in the PR.

## Why

It seems that we don't know why `jsesc` was used in the first place, but it is breaking the markdown rendering.

**Before:**
image

**After:**
image

Todo:

- [x] ~Add tests~
- [ ] Add docs

## Change Type

Indicate the type of change your pull request is:

- [ ] `documentation`
- [x] `patch`
- [ ] `minor`
- [ ] `major`

### CODE REVIEW

File: plugins/microsoft-teams/__tests__/index.test.ts
Comment (mathieubergeron) [inline plugins/microsoft-teams/__tests__/index.test.ts:240]: same thing (https://github.com/intuit/auto/pull/2080/files#r713249678)

File: plugins/microsoft-teams/__tests__/index.test.ts
Comment (mathieubergeron) [inline plugins/microsoft-teams/__tests__/index.test.ts:279]: same thing (https://github.com/intuit/auto/pull/2080/files#r713249678)

File: plugins/microsoft-teams/__tests__/microsoft-teams.test.ts
Comment (mathieubergeron) [inline plugins/microsoft-teams/__tests__/microsoft-teams.test.ts:22]: It looks to me like those 2 tests are identical to the ones in `index.test.ts`:

- "should call Microsoft Office Teams api with minimal config"
- "should call Microsoft Office Teams api through http proxy"

**Comment by mathieubergeron** [comment]
ping @reintroducing @vincentbriglia @hipstersmoothie 

Let me know is you see anything wrong, thank you!! :)

**Review by jakobe — APPROVED** [review]
LGTM!

**Comment by mathieubergeron** [comment]
@jakobe Thanks! Is the coverage check (codecov/project) blocking this PR from getting merged?

**Comment by mathieubergeron** [comment]
ping @jakobe 

Can we get this merged? Thanks!

**Comment by jakobe** [comment]
> ping @jakobe
> 
> Can we get this merged? Thanks!

Hey @mathieubergeron  sorry for the waaaaay late reply, I have a mail filter setup for Github mails otherwise I just drown 😅 , so haven't paid attention to this 🙈 

I'm not the maintainer of this project, just chimed in with my findings and a PR approval to state that it looks good IMHO 👍🏻 

ping @hipstersmoothie 😄

**Comment by jakobe** [comment]
> > ping @jakobe
> > Can we get this merged? Thanks!
> 
> Hey @mathieubergeron sorry for the waaaaay late reply, I have a mail filter setup for Github mails otherwise I just drown 😅 , so haven't paid attention to this 🙈
> 
> I'm not the maintainer of this project, just chimed in with my findings and a PR approval to state that it looks good IMHO 👍🏻
> 
> ping @hipstersmoothie 😄

@hipstersmoothie Any plans for merging this? 🙏🏻

**Comment by adierkens** [comment]
:rocket: PR was released in [`v10.32.3`](https://github.com/intuit/auto/releases/tag/v10.32.3) :rocket:

**Comment by jakobe** [comment]
Thx @hipstersmoothie 🙏🏻

**Pull request merged**
